src/js/components/AppComponent.jsx

```diff
@@ -13,18 +13,17 @@ var statusNameMapping = {
 var AppComponent = React.createClass({
   displayName: "AppComponent",
 
+  contextTypes: {
+    router: React.PropTypes.func
+  },
+
```

Comment:
*FTR*: We are using the `this.context.router` instead of the [Navigation Mixin](http://rackt.github.io/react-router/#Navigation) to enable testing. `contextTypes` is a not-so-documented part of React that react-router uses "[behind the scenes](https://github.com/rackt/react-router/blob/v0.13.3/modules/createRouter.js#L522)" to give components an access to the "global" `router` instance.